.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "go.buildTags": "all smoke",
```

Comment:
This stopped "run test" widgets working in non-smoke test files

Reply:
I have a PR to fix this - I think I've just forgotten to merge it!

Reply:
Queued: https://github.com/pulumi/pulumi/pull/10879

Reply:
If that fails to merge for any reason, I'll pull the change into a separate PR.